shell/probe/nocturnal.sh

```diff
@@ -1,30 +1,33 @@
 #!/bin/bash
 
+PRELUDE_SLEEP=${PRELUDE_SLEEP:=14440}
+PRELUDE_DIR=${PRELUDE_DIR:="./cache"}
```

Comment:
could we at least call this somethign more prelude-specific?

Reply:
yea, still thinking on this new variable overall.. got a name suggestion vs cache?

Reply:
im bad at names ha but presumably something including the work "prelude" unless there's a reason you're avoiding it